CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [0.22.2] - 2002.09.08
```

Comment:
Actually we are not in 2002 ^^  I'll push a fix.

Reply:
Oops 👀